python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -809,11 +828,12 @@ def run(
                 psf_stacker = psf_stacker_gc[cellInfo.index]
                 psf_stacker.add_masked_image(warped_psf_maskedImage, weight=weight)
 
-                if warped_psf_maskedImage.image.array.sum() < 0.995:
+                if not (0.995 < (psf_normalization := warped_psf_maskedImage.image.array.sum()) < 1.005):
```

Comment:
I think at some point this needs to actually do something rather than warn, like try to increase the padding and then fall back to renormalizing by the sum (which should only happen if the PSF cut off, i.e. >0 at the edge).